images/vnc/vnc/index.html

```diff
@@ -38,7 +38,7 @@
         // ex. /api/v0/workspaces/[id]/proxy/apps/desktop/)
         path = window.location.pathname;
         path = path.replace(/^\/|\/$/g, "");
-        path = path + "/websockify";
+        path = path + "websockify";
```

Comment:
I did not test this, but we can handle this more generically.
```js
path = path.replace(/\/$/, '') + "/websockify"
```

Stole from this gist: https://gist.github.com/jens1101/9f3faa6c2dae23537257f1c3d0afdfdf